Uplift/Controllers/HomeViewController.swift

```diff
@@ -40,21 +41,25 @@ class HomeViewController: UIViewController {
         static let lookingForListCellIdentifier = "lookingForListCellIdentifier"
         static let todaysClassesListCellIdentifier = "todaysClassesListCellIdentifier"
         static let todaysClassesEmptyCellIdentifier = "todaysClassesEmptyCellIdentifier"
+        static let sportsListCellIdentifier = "sportsListCellIdentifier"
     }
 
     // MARK: - Enums
     enum SectionType: String {
         case checkIns = "DAILY CHECK-INS"
         case myGyms = "MY GYMS"
         case todaysClasses = "TODAY'S CLASSES"
+        case yourSports = "Your Sports"
```

Comment:
Should this be capitalized?

Reply:
on the figma it wasn't but tbh it does look a little funny so I'll change it